code/modules/mob/living/carbon/human/species_types/grod.dm

```diff
@@ -0,0 +1,396 @@
+	speak_emote = list("squeaks")
+	color = "#00FF00"
+	can_be_held = TRUE
+	worn_slot_flags = ITEM_SLOT_HEAD
+	head_icon = 'icons/mob/species/grod/crown_spider_worn.dmi'
+	held_state = "crown_spider"
+	initial_language_holder = /datum/language_holder/grodcrown //They can only speak Poh'lan in spider
+	var/datum/mind/origin
+
+	pass_flags = PASSMOB
+	speed = 3
+	can_be_held = TRUE
+
+/mob/living/simple_animal/hostile/crown_spider/UnarmedAttack(atom/M)
+	if(try_infect(M))
+		return
+	..()
+
+/mob/living/simple_animal/hostile/crown_spider/MouseDrop(over)
+	. = ..()
+	try_infect(over)
+
+/mob/living/simple_animal/hostile/crown_spider/proc/try_infect(atom/target) //default process before action
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.stat > CONSCIOUS && (alert("This one is no longer living. Are you sure we should infest it?",,"Yes", "No") == "No"))
+			return
+		if(H.getorganslot(ORGAN_SLOT_BRAIN))
+			to_chat(src, "<span class='userdanger'>A foreign presence repels us from this body. Perhaps we should try to infest another?</span>")
+			return
+		Infect(H)
+		return TRUE
+
+	else if(istype(target, /obj/machinery/hydroponics))
+		var/obj/machinery/hydroponics/O = target
+		for(var/obj/item/organ/brain/grod/brain in src)
+			if(O.myseed)
+				to_chat(src, "<span class='userdanger'>This tray is already in use!</span>")
+				return
+			O.myseed = brain.seed
+			src.visible_message("<span class='danger'>[src] burrows into the hydroponics tray!</span>", "<span class='danger'>You borrow into the hydroponics tray, attempting to grow a new body!</span>")
+			qdel(src)
+		return TRUE
+
+/mob/living/simple_animal/hostile/crown_spider/proc/Infect(mob/living/carbon/C)
+	if(!origin)
+		return
+	for(var/obj/item/organ/I in src)
+		I.Insert(C, 1)
+		//Deal a minimum of 30 damage, if damage dealt is greater than 115, (116 will kill a brain), the amount of damage dealt will taper off until it's safe.
+		//So 30 damage_to_deal at 90 I.damage would taper down to 25
+		var/damage_to_deal = (maxHealth - health) * 4 > CROWNSPIDER_MAX_HEALTH ? (maxHealth - health) * 4 : CROWNSPIDER_MAX_HEALTH
+		damage_to_deal = ((damage_to_deal+I.damage) < 115) ? damage_to_deal : damage_to_deal - ((damage_to_deal+I.damage)-115)
+		I.damage += damage_to_deal
+	if(!isgrod(C)) //Convert non-grod hosts to grods over time
+		C.ForceContractDisease(new /datum/disease/transformation/grod())
+	announce_infest(C)
+	origin.transfer_to(C)
+	C.key = origin.key
+	qdel(src)
+
+/mob/living/simple_animal/hostile/crown_spider/proc/announce_infest(mob/living/carbon/target)
+	if(isgrod(target))
+		src.visible_message("<span class='danger'>[src] burrows into [target], planting itself firmly into [target.p_their()] head!</span>")
+	else
+		src.visible_message("<span class='danger'>[src] burrows into [target]'s head!</span>")
+
+/obj/structure/grod_caccoon
+	name = "grod caccoon"
+	desc = "A mysterious phenominom, rarely observed." //fix spilling plox
```

Comment:
bro, can Kapu not google?

phenomenon

also this desc is not very good

Reply:
I didn't write this asshat, use git blame.